addons/wp/XEH_preInit.sqf

```diff
@@ -6,17 +6,17 @@ ADDON = false;
 if (isServer) then {
     [QGVAR(RegisterArtillery), {
         {
-            private _artillery = [GVAR(SideArtilleryHash), side _x] call CBA_fnc_hashGet;
+            private _artillery = GVAR(SideArtilleryHash) get (side _x);
```

Comment:
For safety i would suggest `getOrDefault` instead of a simple get.

Reply:
Yeah true and default should be a empty array 👍